.github/workflows/gpt_pullrequest_updater.yml

```diff
@@ -0,0 +1,50 @@
+name: GPT Pull Request Updater
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  update_pull_request:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19
+
+      - name: Checkout GPT-PullRequest-Updater
+        uses: actions/checkout@v2
+        with:
+          repository: ravilushqa/gpt-pullrequest-updater
+          path: gpt-pullrequest-updater
+
+      - name: Build description and review commands
+        run: |
+          cd gpt-pullrequest-updater
+          go build -o description ./cmd/description
+          go build -o review ./cmd/review
+
+      - name: Update Pull Request Description
+        run: |
+          ./gpt-pullrequest-updater/description
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_TOKEN: ${{ secrets.OPENAI_TOKEN }}
+          OWNER: ${{ github.repository_owner }}
+          REPO: ${{ github.event.repository.name }}
+          PR_NUMBER: ${{ github.event.number }}
+
+      - name: Review Pull Request
+        if: github.event.action == 'opened'
+        run: |
+          ./gpt-pullrequest-updater/review
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_TOKEN: ${{ secrets.OPENAI_TOKEN }}
+          OWNER: ${{ github.repository_owner }}
+          REPO: ${{ github.event.repository.name }}
+          PR_NUMBER: ${{ github.event.number }}
```

Comment:
Overall, the code patch looks good. Here are some suggestions:

- In the "Set up Go" step, it might be helpful to specify a more specific Go version instead of using "1.19", which could become outdated.
- Instead of hardcoding the repository owner and name in the "Checkout GPT-PullRequest-Updater" step, consider using the predefined "github" context variables. For example, you can use "github.repository_owner" and "github.event.repository.name" to get the owner and name of the repository where the pull request was opened/synchronized.
- In the "Update Pull Request Description" and "Review Pull Request" steps, it might be a good idea to add some error handling and logging to make troubleshooting easier if something goes wrong.
- Consider adding some comments to explain what each step does and why it's needed. This can be helpful for other contributors who might not be familiar with the codebase.